frontend/frontend/src/components/SignInSignUpForm.jsx

```diff
@@ -9,20 +9,6 @@ import logo from "../assets/OLIMPlogo.png";
 
 const SignInSignUpForm = ({ closeFormCallback }) => {
   const textSize = "text-[12px] md:text-[14px] lg:text-[16px] xl:text-[18px]";
-  const fields = [
-    {
-      title: "Username",
-      type: "text",
-      placeholder: "Enter your username",
-      icon: <FiUser />,
-    },
-    {
-      title: "Password",
-      type: "password",
-      placeholder: "Enter your password",
-      icon: <RiLockPasswordLine />,
-    },
-  ];
```

Comment:
maybe just add onChangeHandler for each field like this:
const fields = [
      {
        title: "Username",
        type: "text",
        placeholder: "Enter your username",
        icon: <FiUser />,
        onChangeHandler: (e) => setUsername(e.target.value),
      },
      {
        title: "Password",
        type: "password",
        placeholder: "Enter your password",
        icon: <RiLockPasswordLine />,
        onChangeHandler: (e) => setPassword(e.target.value),
      },
    ];